site/gdocs/components/OwidGdocHeader.tsx

```diff
@@ -114,10 +114,10 @@ function OwidArticleHeader({
 function OwidTopicPageHeader({ content }: { content: OwidGdocPostContent }) {
     return (
         <header className="topic-page-header grid span-cols-14 grid-cols-12-full-width">
-            <h1 className="display-1-semibold col-start-2 span-cols-8">
+            <h1 className="display-1-semibold col-start-2 span-cols-8 col-sm-start-2 span-sm-cols-12">
```

Comment:
```suggestion
            <h1 className="display-1-semibold col-start-2 span-cols-8 span-sm-cols-12">
```

I assume we don't need that rule, right?

Reply:
Okay nvm, I can see now that we actually need it